Lib/bdb.py

```diff
@@ -574,6 +574,8 @@ def format_stack_entry(self, frame_lineno, lprefix=': '):
             line = linecache.getline(filename, lineno, frame.f_globals)
             if line:
                 s += lprefix + line.strip()
+        else:
+            print('bdb.Bdb.format_stack_entry: lineno is None')
```

Comment:
Maybe we should add the string to `s` instead of printing it in `bdb`. Adding a `Warning:` prefix would be nice too.